libs/barista-components/alert/src/alert.ts

```diff
@@ -29,12 +38,20 @@ import {
     role: 'alert',
     class: 'dt-alert',
     '[class.dt-alert-warning]': 'severity === "warning"',
+    '[class.dt-alert-info]': 'severity === "info"',
   },
   preserveWhitespaces: false,
   changeDetection: ChangeDetectionStrategy.OnPush,
   encapsulation: ViewEncapsulation.Emulated,
 })
 export class DtAlert {
   /** The severity type of the alert. */
-  @Input() severity: 'error' | 'warning' = 'error';
+  @Input() severity: AlertSeverityType = 'error';
```

Comment:
Please don't put the union type in a type declaration. if you inline it the consumer has better IDE support for the value of the input.

```suggestion
  @Input() severity: 'error' | 'warning' | 'info' = 'error';
```

Reply:
Fixed